code/modules/research/xenobiology/crossbreeding/stabilized.dm

```diff
@@ -23,44 +22,37 @@ Stabilized extracts:
 	qdel(linked_effect)
 	return ..()
 
+/// Returns the mob that is currently holding us if we are either in their inventory or a backpack analogue
+/obj/item/slimecross/stabilized/proc/get_held_mob()
+	if(!loc)
```

Comment:
```suggestion
	if(isnull(loc))
```

especially nicer to use this here since we can run this every single `process()`